ui/package.json

```diff
@@ -14,6 +14,7 @@
     "react-dom": "^17.0.2",
```

Comment:
this package is not used in UI project. Please cleanup changes from UI.

Reply:
done :)